### DISCUSSION TITLE
Support for timeseries forecasting?

### DISCUSSION THREAD

**Discussion by shevm** [oraios/sensAI#91] [Q&A] [answered]
Are there plans to add support for time series forecasting in future?

**Comment by schroedk** [comment, marked as answer]
Hey @shevm, sensAI already provides abstractions for time series forecasting. There is a WIP branch [dengai](https://github.com/opcode81/sensAI/tree/dengai), which shows the usage of sensAI for a time series problem [example](https://github.com/opcode81/sensAI/tree/dengai/examples/dengue-fever-time-series). It is still work in progress and not fully documented. Let us know, if this is helpful for you.

**Comment by shevm** [comment]
Thanks @schroedk! Indeed very helpful. I will have a look at the example.

**Reply by opcode81** [reply]
@shevm I have now merged the example to the current `develop` branch, adding the dataset and a README file, which you may find helpful. The examples can now directly be run.
https://github.com/opcode81/sensAI/tree/develop/examples/dengue-fever-time-series

If you have questions, don't hesitate to reach out.
